includes/admin/reporting/reports.php

```diff
@@ -1038,10 +1077,11 @@ function edd_register_refunds_report( $reports ) {
 			'label' => __( 'Average Time to Refund', 'easy-digital-downloads' ),
 			'views' => array(
 				'tile' => array(
-					'data_callback' => function () use ( $dates ) {
+					'data_callback' => function () use ( $dates, $currency ) {
 						$stats = new EDD\Stats();
 						return $stats->get_average_refund_time( array(
-							'range' => $dates['range'],
+							'range'    => $dates['range'],
+							'currency' => $currency
```

Comment:
I'm not sure that the `get_average_refund_time` method is taking the currency into account. I've got orders in two different currencies and have issued some refunds in just one of them. When I filter by the other one, all of the tiles are showing 0/-- except for this one, which is still showing the overall average refund time (unchanged from the other views).

Reply:
I'll investigate this tomorrow.

Reply:
Fixed via https://github.com/easydigitaldownloads/easy-digital-downloads/pull/8676/commits/6ea2552f8da0845f5a13809a4a997167f602b5bb